app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -32,27 +50,429 @@ var RepositoryDatatable = (function(global) {
+      var $row;
+      var data;
+      var rowId;
+      var index;
+
+      if (currentMode !== 'viewMode') {
+        return;
+      }
+      // Get row ID
+      $row = $(this).closest('tr');
+      data = TABLE.row($row).data();
+      rowId = data.DT_RowId;
+
+      // Determine whether row ID is in the list of selected row IDs
+      index = $.inArray(rowId, rowsSelected);
+
+      // If checkbox is checked and row ID is not in list of selected row IDs
+      if (this.checked && index === -1) {
+        rowsSelected.push(rowId);
+      // Otherwise, if checkbox is not checked and row ID is in list of selected row IDs
+      } else if (!this.checked && index !== -1) {
+        rowsSelected.splice(index, 1);
+      }
+
+      if (this.checked) {
+        $row.addClass('selected');
+      } else {
+        $row.removeClass('selected');
+      }
+
+      updateDataTableSelectAllCtrl();
+
+      e.stopPropagation();
+      updateButtons();
+      // Update number of selected records info
+      $('#selected_info').html(' (' + rowsSelected.length + ' entries selected)');
+    });
+
+    // Handle click on "Select all" control
+    $('.dataTables_scrollHead input[name="select_all"]').change(function(e) {
+      if (this.checked) {
+        $('.repository-row-selector:not(:checked)').trigger('click');
+      } else {
+        $('.repository-row-selector:checked').trigger('click');
+      }
+      // Prevent click event from propagating to parent
+      e.stopPropagation();
+    });
+  }
+
+  function appendInput(form, val, name) {
+    $(form).append(
+      $('<input>').attr('type', 'hidden').attr('name', name).val(val)
+    );
+  }
+
+  // Takes object and surrounds it with input
+  function changeToInputField(object, name, value) {
+    return "<div class='form-group'><input class='form-control' data-object='"
+        + object + "' name='" + name + "' value='" + value + "'></input></div>";
+  }
+
+  // Takes object and surrounds it with input
+  function changeToInputFileField(object, name, value) {
+    return "<div class='repository-input-file-field'>"
+        + "<div class='new-input-file-field-div'><div class='form-group'>"
+        + "<input type='file' class='form-control' data-object='"
+        + object + "' name='" + name + "' value='" + value + "'></div>"
+        + "<a onClick='clearFileInput(this)'>"
+        + "<i class='fas fa-times'></i>"
+        + '</a></div></div>';
+  }
+
+  function initHeaderTooltip() {
+    // Fix compatibility of fixed table header and column names modal-tooltip
+    $('.modal-tooltip').off();
+    $('.modal-tooltip').hover(function() {
+      var $tooltip = $(this).find('.modal-tooltiptext');
+      var offsetLeft = $tooltip.offset().left;
+      var offsetTop = $tooltip.offset().top;
+      if ((offsetLeft + 200) > $(window).width()) {
+        offsetLeft -= 150;
+      }
+      $('body').append($tooltip);
+      $tooltip.css('background-color', '#d2d2d2');
```

Comment:
I think better move it to CSS file and here simply add unique class.

Reply:
This is some hack for datatables